pydoctor/sphinx_ext/build_apidocs.py

```diff
@@ -155,6 +156,6 @@ def setup(app: Sphinx) ->  Mapping[str, Any]:
 
     return {
         'version': __version__,
-        'parallel_read_safe': True,
-        'parallel_write_safe': True,
+        'parallel_read_safe': False,
+        'parallel_write_safe': False,
```

Comment:
Is this change still needed?

Reply:
is not needed yet ... I will revert.  Will be needed when we will introduce the `{source_reference}` placeholder. But I can add it there. Thanks!